pkg/hintrunner/zero/zerohint.go

```diff
@@ -146,7 +146,7 @@ func GetHintFromCode(program *zero.ZeroProgram, rawHint zero.Hint, hintPC uint64
 	case allocSegmentCode:
 		return createAllocSegmentHinter(resolver)
 	case vmEnterScopeCode:
-		return createVMEnterScopeHinter(resolver)
+		return createMemcpyEnterScopeHinter(resolver)
```

Comment:
this is the correct of doing it, but you need to add the hint code as separate code. vmEnterScopeCode sure enters the scope, but it's not the exact code you want. So just add to the codes:
```memcpyEnterScope string = "vm_enter_scope({'n': ids.len})"```
and replace vmEnterScopeCode with the memcpyEnterScope in this case.

Reply:
Oops, that slipped through! I've fixed it now.

Reply:
I see, thanks for the explanation. This is fixed now!